go.mod

```diff
@@ -2,4 +2,23 @@ module github.com/appleboy/gofight/v2
 
 go 1.13
 
-require github.com/stretchr/testify v1.4.0
+require (
+	gitea.com/lunny/tango v0.6.5
+	github.com/andybalholm/brotli v1.0.1 // indirect
+	github.com/astaxie/beego v1.12.3
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gin-gonic/gin v1.6.3
+	github.com/gofiber/fiber/v2 v2.3.3
+	github.com/gorilla/context v1.1.1 // indirect
+	github.com/gorilla/mux v1.8.0
+	github.com/gorilla/pat v1.0.1
+	github.com/julienschmidt/httprouter v1.2.0
+	github.com/klauspost/compress v1.11.7 // indirect
+	github.com/kr/text v0.2.0 // indirect
+	github.com/labstack/echo/v4 v4.1.17
+	github.com/stretchr/testify v1.7.0
+	github.com/valyala/fasthttp v1.19.0 // indirect
+	golang.org/x/sys v0.0.0-20210113181707-4bcb84eeeb78 // indirect
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+)
```

Comment:
revert these changes.

Reply:
1.To add fiber support might have to add fiber itself into the go module file, since fiber server is based on the fasthttp, which is different from the golang's original net/http package. 
Please see following reference for performance benchmarking.
fasthttp: [(https://github.com/valyala/fasthttp)](https://github.com/valyala/fasthttp)
fiber: [(https://github.com/gofiber/fiber)](https://github.com/gofiber/fiber)
2. It would be better to keep the version of beego used in testing indirectly to keep the test working.

Reply:
Yes, I know your point. I remove all packages from `go.mod` since renaming the `example` to `_example`. So we don't need all the framework package if you want to use the package.

Reply:
See the reference: https://github.com/appleboy/gofight/pull/81

Reply:
Any feedbacks?